### PR TITLE
Add fallback values for BMH details page

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx [hunk 1]
```diff
@@ -73,8 +73,10 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
   const nodeName = getMachineNodeName(machine);
   const node = getMachineNode(machine, nodes);
   const role = getMachineRole(machine);
-  const RAMGB = humanizeDecimalBytes(getHostRAMMiB(host) * 2 ** 20).string;
-  const totalStorageCapacity = humanizeDecimalBytes(getHostTotalStorageCapacity(host)).string;
+  const hostRAM = getHostRAMMiB(host);
+  const RAMGB = hostRAM ? humanizeDecimalBytes(hostRAM * 2 ** 20).string : '-';
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx:77]: There is a `DASH` constant in @console/shared

```suggestion
  const RAMGB = hostRAM ? humanizeDecimalBytes(hostRAM * 2 ** 20).string : DASH;
```

same for the other dashes below

File: frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx [hunk 2]
```diff
@@ -185,9 +188,15 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
...
-                    <DetailPropertyListItem title="Vendor">{bios.vendor}</DetailPropertyListItem>
-                    <DetailPropertyListItem title="Date">{bios.date}</DetailPropertyListItem>
+                    <DetailPropertyListItem title="Version">
+                      {bios.version || DASH}
+                    </DetailPropertyListItem>
+                    <DetailPropertyListItem title="Vendor">
+                      {bios.vendor || DASH}
+                    </DetailPropertyListItem>
+                    <DetailPropertyListItem title="Date">
+                      {bios.date || DASH}
```
Comment (mareklibra) [inline frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx:198]: Shouldn't this be humanized?

Comment (rawagner) [reply frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx:198]: out of scope for this PR, but I will verify once there's some baremetal env available

Comment (mareklibra) [reply frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx:198]: sounds good

**Pull request merged**